common/djangoapps/student/management/commands/bulk_change_enrollment.py

```diff
@@ -60,43 +67,62 @@ class Command(BaseCommand):
 
     def handle(self, *args, **options):
         course_id = options.get('course')
+        org = options.get('org')
         from_mode = options.get('from_mode')
         to_mode = options.get('to_mode')
         commit = options.get('commit')
 
-        if course_id is None:
-            raise CommandError('No course ID given.')
+        if (not course_id and not org) or (course_id and org):
+            raise CommandError('You must provide either a course ID or an org, but not both.')
+
         if from_mode is None or to_mode is None:
             raise CommandError('Both `from` and `to` course modes must be given.')
 
-        try:
-            course_key = CourseKey.from_string(course_id)
-        except InvalidKeyError:
-            raise CommandError('Course ID {} is invalid.'.format(course_id))
-
-        if modulestore().get_course(course_key) is None:
-            raise CommandError('The given course {} does not exist.'.format(course_id))
-
+        course_keys = []
+        if course_id:
+            try:
+                course_key = CourseKey.from_string(course_id)
+            except InvalidKeyError:
+                raise CommandError('Course ID {} is invalid.'.format(course_id))
+
+            if modulestore().get_course(course_key) is None:
+                raise CommandError('The given course {} does not exist.'.format(course_id))
+
+            course_keys.append(course_key)
+        else:
+            course_keys = [course.id for course in CourseOverview.get_all_courses(orgs=[org])]
+            if not course_keys:
+                raise CommandError('No courses exist for the org "{}".'.format(org))
+
+        for course_key in course_keys:
+            self.move_users_for_course(course_key, from_mode, to_mode, commit)
+
+        if not commit:
+            logger.info('Dry run, changes have not been saved. Run again with "commit" argument to save changes')
+
+    def move_users_for_course(self, course_key, from_mode, to_mode, commit):
+        """
+        Change the enrollment mode of users for a course.
+        course_key (CourseKey): to lookup the course.
+        from_mode (str): the enrollment mode to change.
+        to_mode (str): the enrollment mode to change to.
+        commit (bool): required to make the change to the database. Otherwise
+               just a count will be displayed.
```

Comment:
This should be:
```
"""
Change the enrollment mode of users for a course.

Arguments:
    course_key (CourseKey): to lookup the course.
    from_mode (str): the enrollment mode to change.
    to_mode (str): the enrollment mode to change to.
    commit (bool): required to make the change to the database. Otherwise
                             just a count will be displayed.
"""
```